CoverMyDotNet/Models/Common/ActionAttributes.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+using RestSharp.Serializers;
+
+namespace CoverMyDotNet
+{
+	public class ActionAttributes
+	{
+		public string Href { get; set; }
+	    public string Ref { get; set; }
```

Comment:
Got some weird indentation here.  Maybe we could add FxCop or something to automatically fix these?